packages/esm-home-app/src/index.ts

```diff
@@ -95,6 +95,14 @@ function setupOpenMRS() {
         online: true,
         offline: false,
       },
+      {
+        id: 'external-app-menu-links',
```

Comment:
Just for my understanding: I thought we now want to use links instead of extensions? Should this rather be a link component or is it so "custom" / "unique" that it fits better as an extension?

Reply:
Well yes, I get your point. The internal ref links lie in the Home app as an extension, so I thought of keeping the external links here too. What do you suggest in here?